source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak-CS.md

```diff
@@ -156,126 +168,132 @@ The JSON response will look something like this:
 You can optionally send and receive XML if you set the `Content-Type` to
 `application/xml`, as in this example:
 
-Once the admin user exists, you must specify the credentials of the
-admin user on each node in the Riak CS system. The admin user credential
-settings reside in the Riak CS `app.config` file, which is located in
-the `etc/riak-cs` directory. The settings appear in the Riak CS config
-section of the file. Paste the `key_id` string between the quotes for
-the `admin_key`. Paste the `key_secret` string into the `admin_secret`
-variable, as shown here:
-
-```appconfig
-%% Admin user credentials
-{admin_key, "324ABC0713CD0B420EFC086821BFAE7ED81442C"},
-{admin_secret, "5BE84D7EEA1AEEAACF070A1982DDA74DA0AA5DA7"},
+Once the admin user exists, you must specify the credentials of the admin user
+on each node in the Riak CS system. The admin user credential settings reside in
+the Riak CS `riak-cs.conf` file, which is located in the `/etc/riak-cs`
+directory. Paste the `key_id` string between the quotes for the `admin.key`.
+Paste the `key_secret` string into the `admin.secret` variable, as shown here:
+
+```riakcsconf
+admin.key, 324ABC0713CD0B420EFC086821BFAE7ED81442C},
+admin.secret, 5BE84D7EEA1AEEAACF070A1982DDA74DA0AA5DA7},
 ```
 
-Once the admin user exists, you must specify the credentials of the
-admin user in the `app.config` file. Those will be the same credentials
-that you received as a JSON object when you ran the `POST` request to
-create the user.
+```advancedconfig
+{riak_cs, [
+           %% Admin user credentials
+           {admin_key, "324ABC0713CD0B420EFC086821BFAE7ED81442C"},
+           {admin_secret, "5BE84D7EEA1AEEAACF070A1982DDA74DA0AA5DA7"},
+           %% Other configs
+          ]}
+```
 
 ## Bucket Restrictions
 
-If you wish, you can limit the number of buckets created per user. The
-default maximum is 100. Please note that if a user exceeds the bucket
-creation limit, they are still able to perform other actions, including
-bucket deletion. You can change the default limit using the
-`max_buckets_per_user` parameter in each node's `app.config` file. The
-example configuration below would set the maximum to 1000:
+If you wish, you can limit the number of buckets created per user. The default
+maximum is 100. Please note that if a user exceeds the bucket creation limit,
+they are still able to perform other actions, including bucket deletion. You can
+change the default limit using the `max_buckets_per_user` parameter in each
+node's `advanced.config` file---there is no equivalent configuration for the
+`riak-cs.conf` file. The example configuration below would set the maximum to
```

Comment:
I just want to confirm, these setting are missing from 2.0.0, but will be added to 2.0.1?